pgxmock.go

```diff
@@ -430,7 +432,15 @@ func (c *pgxmock) Query(ctx context.Context, sql string, args ...interface{}) (p
 		if err := c.queryMatcher.Match(queryExp.expectSQL, sql); err != nil {
 			return err
 		}
-		if err := queryExp.argsMatches(args); err != nil {
+		if rewrittenSQL, err := queryExp.argsMatches(sql, args); err != nil {
+			return err
+		} else if rewrittenSQL > "" {
```

Comment:
question (non-blocking): curious why `>` was used instead of `!=`?

Reply:
Old habit. In some old languages and compilers this was faster :-)